src/main/java/com/bio4j/model/uniprot/programs/ImportProteinInteractions.java

```diff
@@ -122,78 +123,106 @@ protected void importProteinInteractions(File inFile, File dbFolder) {
 
         Logger.getLogger(ImportProteinInteractions.class.getName()).log(Level.SEVERE, null, ex);
       }
-
     }
   }
 
-  private void importProteinInteractionsWithSource(
-    XMLElement entryXMLElem,
-    UniProtGraph<I,RV,RVT,RE,RET> graph,
-    Protein<I,RV,RVT,RE,RET> protein
-  )
-  {
-
-    List<Element> comments = entryXMLElem.asJDomElement().getChildren(COMMENT_TAG_NAME);
+  /*
+    By direct inspection of UniProt XML file I have concluded that:
 
-    for (Element commentElem: comments) {
+    1. the source id is always a protein, I don't know why
+    2. target can be either an isoform or a protein, and the id is in `interactant/id`. Isoform ids *look* to be `${protein.id}-{number}`
+    3. the `intactId`s have nothing to do with UniProt, and should be dropped
```

Comment:
They are crossrefs mapping UniProt ids to IntAct ids. As such they don't make much sense living duplicated in an edge; if imported, they should be an arrow (or a property) of a protein.